npm/webpack-dev-server/test/devServer-e2e.spec.ts

```diff
@@ -68,7 +68,7 @@ const cypressConfig = {
   supportFile: '',
   isTextTerminal: true,
   devServerPublicPathRoute: root,
-  indexHtmlFile: path.join(__dirname, 'component-index.html'),
+  indexHtmlFile: 'cypress/support/component-index.html',
```

Comment:
What is the reason for this change?  I am not familiar with this test, but there is a `component-index.html` file in the same directory as this test file that was previously being referenced.  It looks like the test modified below is just checking that the configured index file is being touched after a spec change.  Why the need to add the new `component-index.html` file under `cypress/support` to do this?  Could you just use the existing one in this same directory?  

Also, component testing is not set up for this package, having that file under `cypress/support` might be confusing to our future selves.  If it needs to stay, then a comment in it saying why it is there might be good.  Also, if it does remain in this test, can the existing file in the same directory be removed?

Reply:
I agree I think it would make more sense to keep the files related to the unit test in close proximity so as to not pollute the actual cypress e2e project setup for this package.

Reply:
I changed this because it was an absolute path when it should be a relative path - but yeah my bad, it should just reference the existing component index, good catch.

Reply:
resolved in https://github.com/cypress-io/cypress/pull/25861/commits/260b0c156009681916e32e5fe048119639eb01a1